### PR TITLE
travis: add job testing tier1 tests with openvswitch2.13

### PR DESCRIPTION
**Pull request by ffmancera**
Signed-off-by: Fernando Fernandez Mancera <ffmancera@riseup.net>

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -8,6 +8,10 @@ env:
         - CONTAINER_IMAGE=docker.io/nmstate/centos8-nmstate-dev
           testflags="--test-type integ_tier1 --pytest-args='-x'
               --copr networkmanager/NetworkManager-1.22-git"
+        - CONTAINER_IMAGE=docker.io/nmstate/centos8-nmstate-dev
+          testflags="--test-type integ_tier1 --pytest-args='-x'
+              --copr networkmanager/NetworkManager-1.22-git
+              --customize='dnf remove openvswitch2.11 python3-openvswitch2.11;'"
```
Comment (cathay4t) [inline .travis.yml:14]: Can you explicitly install ovs 2.13 here?

Comment (ffmancera) [reply .travis.yml]: In order to install ovs 2.13, I need to remove 2.11 first. If not it will generate conflicts between the package files.

**Comment by ffmancera** [comment]
After working a lot on this, I am proposing the best way to do it avoiding the expanding from the environment variable. I have try it using `getopt` and `getopts` but it is not working. This way it is allowing us to use customize argument on travis CI. Thanks to @pablombg for all the help!

File: automation/tests-container-utils.sh [hunk 2]
```diff
@@ -78,7 +77,12 @@ function container_pre_test_setup {
     fi
 
     create_container
-    if [[ -v customize_cmd ]];then
+    if [[ -v "$customize_cmd" ]];then
+        if [[ "$CI" == "true" ]];then
+            container_exec "rm -fv /var/cache/dnf/metadata_lock.pid"
+            container_exec "dnf clean all"
+            container_exec "dnf makecache || :"
+        fi
```
Comment (cathay4t) [inline automation/tests-container-utils.sh:85]: Can you wrap this into a function so that it could be used in `upgrade_nm_from_copr()` also?

File: automation/tests-container-utils.sh [hunk 3]
```diff
@@ -1,4 +1,4 @@
-#!/bin/bash -ex
+#!/bin/bash -e
```
Comment (cathay4t) [inline automation/tests-container-utils.sh:1]: This is not related for this PR.

File: automation/tests-container-utils.sh (hunk 2 above)
Comment (ffmancera) [reply automation/tests-container-utils.sh]: Done, thanks!

File: automation/tests-container-utils.sh (hunk 3 above)
Comment (ffmancera) [reply automation/tests-container-utils.sh]: Removed.

**Comment by ffmancera** [comment]
> The customize command is not actually ran in CI.

Fixed. The CI is now running the customize commands.

**Comment by ffmancera** [comment]
@cathay4t openvswitch 2.13 is being installed correctly now.
